processing_chain/src/main/groovy/org/orbisgis/processingchain/BuildLCZ.groovy

```diff
@@ -234,22 +228,18 @@ public static createLCZ() {
 
                 // III. Define the LCZ of each RSU according to their 7 geometric and surface cover properties
                 // Merge all indicator columns in one table
-                datasource.execute(("DROP TABLE IF EXISTS $rsu_indic1; CREATE TABLE $rsu_indic1 AS SELECT a.*, " +
-                        "b.rsu_aspect_ratio FROM $rsu_indic0 a INNER JOIN " +
-                        "${calc_aspect_ratio.results.outputTableName} b ON a.$columnIdRsu = b.$columnIdRsu").toString())
-                datasource.execute(("DROP TABLE IF EXISTS $rsu_indic2; CREATE TABLE $rsu_indic2 AS SELECT a.*, " +
-                        "b.rsu_ground_sky_view_factor FROM $rsu_indic1 a INNER JOIN " +
-                        "${calcSVF.results.outputTableName} b ON a.$columnIdRsu = b.$columnIdRsu").toString())
-                datasource.execute(("DROP TABLE IF EXISTS $rsu_indic3; CREATE TABLE $rsu_indic3 AS SELECT a.*, " +
-                        "b.pervious_fraction, b.impervious_fraction FROM $rsu_indic2 a INNER JOIN " +
-                        "${calc_perviousness_frac.results.outputTableName} b ON a.$columnIdRsu = b.$columnIdRsu").toString())
-                datasource.execute(("DROP TABLE IF EXISTS $rsu_all_indic; CREATE TABLE $rsu_all_indic AS SELECT a.*, " +
-                        "b.effective_terrain_roughness_class FROM $rsu_indic3 a INNER JOIN " +
-                        "${calc_roughness_class.results.outputTableName} b ON a.$columnIdRsu = b.$columnIdRsu").toString())
+                IProcess join_final_table = Geoclimate.DataUtils.joinTables()
+                join_final_table.execute([inputTableNamesWithId:
+                        [(join_for_SVF.results.outputTableName)       : columnIdRsu,
+                         (calc_aspect_ratio.results.outputTableName)  : columnIdRsu,
+                         (calcSVF.results.outputTableName)  : columnIdRsu,
+                         (calc_perviousness_frac.results.outputTableName)  : columnIdRsu,
+                         (calc_roughness_class.results.outputTableName)  : columnIdRsu]
```

Comment:
Write without parenthesis   calc_roughness_class.results.outputTableName:columnIdRsu

Reply:
I have initially done that but it does not work. It seems I can not write it without parenthesis when it is used as key in a map